src/System.Private.ServiceModel/src/System/ServiceModel/Channels/BufferedWriteStream.cs

```diff
@@ -34,21 +33,25 @@ namespace System.ServiceModel.Channels
     /// </summary>
     internal sealed class BufferedWriteStream : Stream
     {
-        public const int DefaultBufferSize = 4096;
+        public const int DefaultBufferSize = 8192;
```

Comment:
Yes, perf improvement. As we have a synchronous read/write path in streaming scenarios, this reduces the number of times we require 2 threads to do anything. Additionally, the buffers now comes from the BufferManager so if the stream is disposed, this will reduce memory consumption so the larger buffer won't have a negative effect.